src/util/InterfaceController.js

```diff
@@ -15,9 +15,9 @@ class InterfaceController {
      */
     async openChatWindow(chatId) {
         await this.pupPage.evaluate(async chatId => {
-            let chatWid = window.Store.WidFactory.createWid(chatId);
-            let chat = await window.Store.Chat.find(chatWid);
-            await window.Store.Cmd.openChatAt(chat);
+            const chatWid = window.Store.WidFactory.createWid(chatId);
+            const chat = window.Store.Chat.get(chat) || await window.Store.Chat.find(chatWid);
```

Comment:
In `window.Store.Chat.get(chat)`, `chat `was used before it was defined

Reply:
Thank you

Reply:
no working